scene/resources/tile_set.cpp

```diff
@@ -149,15 +149,19 @@ bool TileSet::_set(const StringName &p_name, const Variant &p_value) {
 		}
 	} else if (what == "shape")
 		tile_set_shape(id, 0, p_value);
```

Comment:
This else if should also have a brackets for consistence.

Reply:
Are you saying that the line should have the for loop as well, or are you just talking about the brackets? 
Throughout the Godot source code it seems to be convention to omit brackets for one-liners. Following this convention, I guess I probably should have just omitted the brackets, since the for loop is a one-liner as well, although I felt the nested one-liners could cause readability problems.